app/main.py

```diff
@@ -2,0 +1,33 @@
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+
+        # Додаємо екземпляр до класової змінної
+        if self.health > 0:
+            Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, Health: {self.health},"
+                f" Hidden: {self.hidden}}}")
+
+    def die(self) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
```

Comment:
Removing `self` from the `Animal.alive` list is appropriate when the animal dies. Ensure that this logic is consistent with your application's requirements.